react/Hero/Readme.md

```diff
@@ -0,0 +1,40 @@
+A `Hero` is used to to showcase key marketing messages. It
+is perfect for onboarding an app.
+
+```jsx
+const Hero = require('.').default
+const { Title, Subtitle, Section, Sections, Paragraph, CTA, Icon} = Hero
+const Button = require('../Button').Button
+const { t } = require('../../docs/utils')
+
+const RED = '#f52d2d'
+const PURPLE = '#a75bcb'
+const BLUE = '#2d8af2';
+
+<Hero>
+  <Title>{t('Onboarding.title')}</Title>
```

Comment:
The title doesn't appear in your example, I suppose this is not on purpose, right?

Reply:
No it's true I must find out why. Good catch !

Reply:
done